ui/src/app/applications/components/search-bar.tsx

```diff
@@ -0,0 +1,96 @@
+import * as models from '../../shared/models';
```

Comment:
I'm wondering if this component could be placed in the ui/src/app/applications/components/shared folder instead. where all the other shared components are.

Reply:
Hi @keithchong , thanks for the review and comments. Do you mean `search-bar` component?